src/lib/forms/UserSubmissionAction.ts

```diff
@@ -0,0 +1,51 @@
+import { fail, type Action } from '@sveltejs/kit';
+import { validateToken } from './validateTurnstileToken';
+import { prisma_client } from '$/server/prisma-client';
+import { env } from '$env/dynamic/private';
+import { user_submission_schema } from '$/lib/forms/userSubmissionSchema';
+import { z } from 'zod';
+
+export const UserSubmissionAction: Action = async function ({ request }) {
+	const form = await request.formData();
+	// Validate Turnsile Token
+	const token = form.get('cf-turnstile-response');
+	let { success, error } = await validateToken(token, env.TURNSTILE_SECRET);
+	if (error) {
+		return fail(400, { message: 'Failed Captcha', error });
+	}
+	// Save to DB
+	const data = {
```

Comment:
The data here should be available on `locals.form_data` instead of having to do form.get().

ie 
locals.form_data.body, locals.form_data.name ect